Updated README regarding deprecation (#100)
opensearch-project/opensearch-dsl-py: README.md

![OpenSearch logo](https://github.com/opensearch-project/opensearch-dsl-py/raw/main/OpenSearch.svg)

> **Warning**
> 

> >The OpenSearch high-level Python client (opensearch-dsl-py) will be deprecated with its final release version as 2.1.0. We recommend switching to the OpenSearch Python client ([opensearch-py](https://github.com/opensearch-project/opensearch-py)) which now includes the functionality of opensearch-dsl-py.

Reviewer: ```suggestion
>The OpenSearch high-level Python client (opensearch-dsl-py) will be deprecated after version 2.1.0.  We recommend switching to the OpenSearch Python client ([opensearch-py](https://github.com/opensearch-project/opensearch-py)), which now includes the functionality of opensearch-dsl-py.
```
